include/alpaka/mem/view/ViewSeqRange.hpp

```diff
@@ -0,0 +1,93 @@
+
+#pragma once
+
+#include "alpaka/core/Common.hpp"
+#include "alpaka/dev/DevCpu.hpp"
+#include "alpaka/mem/view/Traits.hpp"
+#include "alpaka/platform/PlatformCpu.hpp"
+
+#include <ranges>
+
+namespace alpaka::trait
+{
+
+    namespace detail
+    {
+        template<typename T>
+        concept ContiguousContainer = requires(T t) {
+            t.size();
+            t.data();
+            requires std::ranges::contiguous_range<T>;
+        };
```

Comment:
```suggestion
        template<typename T>
        concept ContiguousContainer =
            std::ranges::contiguous_range<T> &&
            requires(T t) {
                t.size();
                t.data();
            };
```
can you check if something like this helps with the error you are getting about the constraint depending on itself?

Reply:
Same error